apps/cli/src/commands/packager/importer-v4.ts

```diff
@@ -485,17 +498,44 @@ export class ImporterV4 {
 
   public async import(): Promise<void> {
     await this.unzipPackage();
-    await this.readPackage();
-
-    try {
-      await this.importLocales();
-      await this.importNutrientTables();
-      await this.importAsServedSets();
-      await this.importGlobalFoods();
-      await this.importLocalFoods();
-      await this.importEnabledLocalFoods();
-    } finally {
-      await this.cleanUpPackage();
+    await this.apiClient.baseClient.refresh();
+
+    if (this.options.modulesForExecution === 'all') {
+      await this.readPackage();
+      try {
+        await this.importLocales();
+        await this.importNutrientTables();
+        await this.importAsServedSets();
+        await this.importGlobalFoods();
+        await this.importLocalFoods();
+        await this.importEnabledLocalFoods();
+      } finally {
+        await this.cleanUpPackage();
+      }
+    } else {
+      switch (this.options.modulesForExecution) {
+        case 'locales':
+          // try {
+          //   await this.readLocales();
+          //   await this.importLocales();
+          // } finally {
+          //   await this.cleanUpPackage();
+          // }
+          logger.info('LOCALES - Not implemented');
+          break;
+        case 'images-as-served':
+          try {
+            await this.readAsServedSets();
+            await this.importAsServedSets();
+          } finally {
+            await this.cleanUpPackage();
+          }
+          break;
+        default:
+          throw new Error(
+            `Unexpected module for execution option: ${this.options.modulesForExecution}`
+          );
+      }
```

Comment:
Hi @cheburan, this switch statement looks suspicious -- "modulesForExecution" is probably an array an you're doing a switch on it. Does this work? 😅

Reply:
HI @mechkg , That code you highlighted doesn't look final; it changed since. I added more options to it. 

But the idea was that the users could choose either one specific module or "all", hence the switch that only allows them to choose either "all" or one of the modules (locale, images-as-served, ...). It doesn't allow an array of options atm. It's better to prevent users from executing a random combination of steps/modules. Should CLI allow it?

Reply:
I'll rebase it, think it's a bit muddled with merged stuff from main

Reply:
@cheburan I see, but it needs to allow multiple modules because sometimes one is dependent on the other (local-foods needs as-served and nutrient-tables). These dependencies don't need to be enforced, but allowing several options to be executed in the specified order should be allowed  (e.g. --as-served --nutrient-tables --local-foods)

Reply:
Ok true. Added the ability to execute multiple steps in a user-specified order. Like: 
```
pnpm run cli import-package -c overwrite -m locales nutrients v4 /path/to/the/archived/ARCHIVED_FOLDER.zip
```
all based on this: 
[https://github.com/MRC-Epid-it24/intake24/pull/220/files#diff-d43093ebd3ed9800863ee697acfb479d16d6efc76928a54e2469e0ec3b9c7267R93](url)